docs/content/preview/develop/change-data-capture/using-logical-replication/advanced-topic.md

```diff
@@ -138,12 +138,16 @@ To enable dynamic table addition, perform the following steps:
     ./yb-ts-cli --server_address=<tserverIpAddress:tserverPort> set_flag cdcsdk_publication_list_refresh_interval_secs 120
     ```
 
-1. After you start receiving records from the newly added table in the publication, reset the  `cdcsdk_publication_list_refresh_interval_secs` flag to a high value (for example, 3600 seconds).
+1. After you start receiving records from the newly added table in the publication, reset the  `cdcsdk_publication_list_refresh_interval_secs` flag back to original value (i.e 900 seconds).
 
     ```sh
-    ./yb-ts-cli --server_address=<tserverIpAddress:tserverPort> set_flag cdcsdk_publication_list_refresh_interval_secs 3600
+    ./yb-ts-cli --server_address=<tserverIpAddress:tserverPort> set_flag cdcsdk_publication_list_refresh_interval_secs 900
     ```
 
+{{< note title="Important" >}}
+In the event that a user ends up reducing the value of `cdcsdk_publication_list_refresh_interval_secs`, it is recommended to increase the value of the flag back to its original value once you start receiving changes from the new table.
```

Comment:
Why?

Reply:
For every publication refresh we incur overhead to invalidate the relcache. Reducing the interval for publication refresh would mean we incur this overhead much more frequently.